dbt/include/snowflake/macros/adapters.sql

```diff
@@ -72,14 +72,16 @@
 )
 {% endmacro %}
 
-{% macro snowflake__create_view_as(relation, sql) -%}
+{% macro create_view_as_with_temp_flag(temporary, relation, sql) -%}
```

Comment:
Are there any backwards compatibility issues with renaming this macro? i.e. - does anyone override this in their projects?

Reply:
I specifically picked a name that I didn't think anyone was using anywhere. Is that too naive?

Reply:
I think we want a `snowflake__` prefix here to serve as a namespacing and make sure we don't accidentally overwrite this somewhere else. Also if we ever want to have this be dispatched from core we will need the prefix

Reply:
To tie off this convo, `snowflake__create_view_as` has been retained, so there shouldn't be any backwards compatibility issues if anyone overrides this in their projects.

A new macro named `snowflake__create_view_as_with_temp_flag` was introduced, and it is utilized by `snowflake__create_view_as`.